src/Provider/Telegram.php

```diff
@@ -120,6 +120,11 @@ public function getUserProfile()
         $userProfile->lastName      = $data->get('last_name');
         $userProfile->displayName   = $data->get('username');
         $userProfile->photoURL      = $data->get('photo_url');
+        $username = $data->get('username');
+        if (!empty($username)) {
+            // Only some accounts have usernames
+            $userProfile->profileURL    = "https://t.me/" . $username;
```

Comment:
a little remark: or you use single quotes here (like standard) or double with `"https://t.me/{$username}"`